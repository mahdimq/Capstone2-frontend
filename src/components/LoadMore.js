import React from 'react';
import PropTypes from 'prop-types';
import { StyledLoadMoreBtn } from '../styles/StyledLoadMoreBtn';

function LoadMore({ text, callback }) {
	return (
		<StyledLoadMoreBtn type='button' onClick={callback}>
			{text}
		</StyledLoadMoreBtn>
	);
}

LoadMore.propTypes = {
	text: PropTypes.string,
	callback: PropTypes.func
};
export default LoadMore;
