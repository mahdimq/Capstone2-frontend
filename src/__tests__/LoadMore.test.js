import React from 'react';
import { render } from '@testing-library/react';
import LoadMore from '../components/LoadMore';

it('renders without crashing', function () {
	render(<LoadMore />);
});

it('matches snapshot', function () {
	const { asFragment } = render(<LoadMore />);
	expect(asFragment()).toMatchSnapshot();
});
