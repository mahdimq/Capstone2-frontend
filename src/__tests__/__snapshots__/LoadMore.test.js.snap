// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <button
    class="sc-bdfBwQ dmPsJE"
    type="button"
  />
</DocumentFragment>
`;
